test/datalog_tests/lb_test/src/main.rs

```diff
@@ -0,0 +1,91 @@
+use lb_ddlog::api::HDDlog;
```

Comment:
why write the test in Rust instead of using the CLI?

Reply:
The goal is to illustrate how the D3log runtime (that will be written in Rust) can extract location information from output.